### PR TITLE
Fix iOS closing the overlay 

### PR DESCRIPTION
**Pull request by manolo**
Connected to #96

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-context-menu/98)

### CODE REVIEW

File: test/touch.html [hunk 1]
```diff
@@ -121,6 +131,16 @@
...
         });
 
+        it('should not preventDefault on touch-end after normal tap', done => {
+          makeSoloTouchEvent('touchstart', null, target);
+
+          setTimeout(() => {
+            const e = makeSoloTouchEvent('touchend', null, target);
+            expect(e.defaultPrevented).to.be.false;
+            done();
+          }, 100);
```
Comment (limonte) [inline test/touch.html]: why `100`?

Comment (YuriyVaadin) [reply test/touch.html]: Is it an imitation of normal tap for 1 microsecond?

File: package-lock.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "vaadin-context-menu",
```
Comment (YuriyVaadin) [inline package-lock.json]: Is it needed to change package-lock.json?

File: test/touch.html (hunk 1 above)
Comment (manolo) [reply test/touch.html]: Yes a synthetic tap emulation, actually anything lesser than the timeout that fires the long tap (contextmenu)

File: package-lock.json (hunk 2 above)
Comment (manolo) [reply package-lock.json]: Done

**Comment by limonte** [comment]
:lgtm:

---

Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-context-menu/98#-:-Ku3G993f2wrAWUmMk69:bnfp4nl)*

**Comment by YuriyVaadin** [comment]
:lgtm:

---

Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-context-menu/98#-:-Ku3P9XQ1Zn29nbJOCq0:bnfp4nl)*

File: test/touch.html (hunk 1 above)
Comment (jouni) [reply test/touch.html]: Would be nice to document these in the source code, so we could avoid these kinds of questions and confusion in the future as well.

**Pull request merged**